plugins/org.csstudio.opibuilder.widgets.edm/src/org/csstudio/opibuilder/widgets/edm/figures/EdmSymbolFigure.java

```diff
@@ -40,9 +40,14 @@ protected void paintClientArea(Graphics graphics) {
         if (image != null) {
             // If what were trying to draw is out of bounds, simply draw the full image to
             // fill the area
```

Comment:
I have updated the comments for the different cases so that it should be clearer what will be plotted.
The rectangle is a filled rectangle the colour specified by the widget property 'Display > Background colour'. I have added this to comment. I think by default (i.e. if the user doesn't change anything), this happens to be blue.